cpplint/cpplint.py

```diff
@@ -5897,6 +5898,27 @@ def CheckStdThread(filename, clean_lines, linenum, error):
       error(filename, linenum, 'build/std_thread',
             4,  # 4 = high confidence
             'Please use yb::Thread instead of std::thread in yugabyte product code')
+   
+_RE_PATTERN_INCLUDE_GLOG_API = re.compile(r'^\s*#include\s+<glog/logging.h>')
```

Comment:
Seems like we can convert this to a generic function and pass in the pattern and error message?

Reply:
Mostly we have some extra checks on filename, so we can probably add that as a regex as well. Created a GH item to change the yb functions. https://github.com/yugabyte/yugabyte-db/issues/19643